shared/scripts/web-terminal.sh

```diff
@@ -21,7 +21,7 @@ else
 fi
 
 echo "--> Configuring Nginx proxy for Wetty web terminal"
-sudo tee /etc/nginx/nginx.conf > /dev/null <<"EOF"
```

Comment:
I believe these quotes are necessary to disable parameter substitution. I believe tests are failing because of it.

Reply:
TIL about this. Historically I would just escape the `$`, but this is much cleaner!

Reply:
Ha - I actually had to look it up as well. I had copied and pasted from somewhere else, removed the quotes, and it didn't work. I put them back and it worked, so didn't investigate too much further.